en/docs/ai-gateway/next/gateway-artifacts/mcp-proxy.md

```
@@ -0,0 +1,95 @@
+spec:
+  displayName: Everything
+  version: v1.0
+  context: /everything
+  specVersion: "2025-06-18"
+  upstream:
+    url: http://everything:3001
+  tools: []
+  resources: []
+  prompts: []
+EOF
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)